ocaml/idl/markdown_backend.ml

```diff
@@ -164,7 +175,7 @@ let markdown_section_of_message printer x =
     printer "";
   end
 
-let print_field_table_of_obj printer x =
+let print_field_table_of_obj printer is_class_deprecated is_class_removed x =
```

Comment:
I think we should make these arguments labeled, to add more safety, because they are both bool:
```ocaml
let print_field_table_of_obj printer ~is_class_deprecated ~is_class_removed x
```
And then we can safely call them:
```ocaml
print_field_table_of_obj printer ~is_class_deprecated ~is_class_removed ...
```
(The same applies to every function taking these new arguments.)